MetaMask/metamask-extension
# Validate LavaMoat config in parallel

## Explanation

The LavaMoat policies and allow-scripts configuration are now validated in parallel. They're also run on every PR now, though they're still only required for release candidates and production releases.

This relates to #19680

## Manual Testing Steps

No function changes, CI only.

This can't be tested directly at the moment because it only gets triggered for release candidates and production releases, but I did temporarily comment out that filter to ensure it worked. You can see that CI run [here](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/49260/workflows/33d98a6e-e497-4691-a2be-7094dbc3ce24). See that the policy validation passed except for the MMI policy, which is fixed in [this PR](https://github.com/MetaMask/metamask-extension/pull/19683).

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
